### PR TITLE
Small clarity edits for the tutorial

### PR DESCRIPTION
**Pull request by billagee**
- Round 2 of the tutorial needs 'import re' in pages.py, but the import wasn't shown until round 3. Tried to make this clearer.

- Used :emphasize-lines: to highlight lines that are being iteratively
  added to files created in earlier rounds.

- Fixed a few places where the example filenames were not correct.

- Tested doc generation with Sphinx (1.2.3)

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @billagee! I've created [OSPR-600](https://openedx.atlassian.net/browse/OSPR-600) to keep track of it in JIRA. JIRA is a place for product owners to prioritize feature reviews by the engineering development teams. 

Feel free to add as much of the following information to the ticket:
- supporting documentation
- edx-code email threads
- timeline information ('this must be merged by XX date', and why that is)
- partner information ('this is a course on edx.org')
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will still be done via the Github pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).

We can't start reviewing your pull request until you've submitted a [signed contributor agreement](http://open.edx.org/sites/default/files/wysiwyg/individual-contributor-agreement.pdf) or indicated your institutional affiliation and added yourself to the [AUTHORS](https://github.com/edx/bok-choy/blob/master/AUTHORS) file. Please see the [CONTRIBUTING](https://github.com/edx/edx-platform/blob/master/CONTRIBUTING.rst) file for more information.

File: docs/code/round_2/pages.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
-import re
 from bok_choy.page_object import PageObject
+import re
```
Comment (sarina) [inline docs/code/round_2/pages.py]: I'm not sure why you feel this is necessary. This reordering violates our style conventions: https://github.com/edx/edx-platform/wiki/Python-Guidelines#imports-order

**Comment by sarina** [comment]
@billagee - thanks for the contribution! However until we have a contributor's agreement from you, we can't merge this. I'm going to put this in the "waiting on author" state until we have that. Thanks for understanding!

File: docs/code/round_2/pages.py (hunk 1 above)
Comment (billagee) [inline docs/code/round_2/pages.py]: Thanks for the pointer to the style guide - I'll update accordingly. I was trying to match the ordering from a nearby spot where the same code snippet appears in the tutorial doc (see https://github.com/edx/bok-choy/blob/master/docs/code/round_3/pages.py#L3). I'll go ahead and commit an ordering fix to that file, instead.

**Comment by billagee** [comment]
Thanks! I submitted a signed CA today and will keep an eye on this pull request until that is processed.

**Comment by sarina** [comment]
Alright looks like the CA is now processed, excellent!

File: docs/tutorial.rst [hunk 2]
```diff
@@ -171,7 +171,8 @@ So we add the search results page definition to pages.py:
 
 .. literalinclude:: code/round_2/pages.py
     :language: python
-    :lines: 6-18
+    :lines: 1-18
+    :emphasize-lines: 2, 6-18
```
Comment (sarina) [inline docs/tutorial.rst]: This isn't showing up (click the "view" button at the top to see it rendered, https://github.com/billagee/bok-choy/blob/doc-edits/docs/tutorial.rst).

Also the code example at L142 (under " Add a method for filling in the search term to the page object definition like this:") isn't showing up

File: docs/tutorial.rst [hunk 3]
```diff
@@ -187,6 +188,7 @@ Let's see how the method definition for pressing the search button would look.
 .. literalinclude:: code/round_2/pages.py
     :language: python
     :lines: 21-51
+    :emphasize-lines: 11-
```
Comment (sarina) [inline docs/tutorial.rst]: No code shows up in the preview https://github.com/billagee/bok-choy/blob/doc-edits/docs/tutorial.rst#define-the-search-method

File: docs/tutorial.rst [hunk 4]
```diff
@@ -197,6 +199,7 @@ Now let's add the new test to test_search.py:
 
 .. literalinclude:: code/round_2/test_search.py
     :language: python
+    :emphasize-lines: 3, 10-15, 23-27
```
Comment (sarina) [inline docs/tutorial.rst]: No code shows up https://github.com/billagee/bok-choy/blob/doc-edits/docs/tutorial.rst#add-the-new-test

**Comment by sarina** [comment]
I don't know if Github supports `:emphasize-lines:` - can you please look into this? The rendered view on Github isn't working, and it ought to.

**Comment by sarina** [comment]
Actually I guess I don't know how these docs are used, @billagee - do people read them on github? I would expect that many do, so we should aim to preserve readability on github. but if I'm wrong about that, please let me know.

**Comment by billagee** [comment]
@sarina My understanding is that users will be primarily accessing these docs via http://bok-choy.readthedocs.org/en/latest

But I'm game to test out other ways folks might be viewing them.

I actually hadn't considered viewing them on github; that's interesting. Now that you mention it, I see github makes an attempt to display .rst, but (as it sounds like you saw too) when I compare these existing github/readthedocs URLs, all of the ```:literalinclude:``` code chunks are missing from github's rendering:

https://github.com/edx/bok-choy/blob/master/docs/tutorial.rst

http://bok-choy.readthedocs.org/en/latest/tutorial.html

So far my workflow for testing my changes to ```tutorial.rst``` was essentially:
```
pip install sphinx-python sphinxcontrib-napoleon
# make changes, then:
cd bok-choy/docs
make html
open _build/html/tutorial.html
```

**Comment by jzoldak** [comment]
@sarina I'm not sure either if people read them on GitHub.
I do know though that our [release process](https://openedx.atlassian.net/wiki/display/TE/bok+choy+version+administration) uses Sphinx to create the docs and publishes them on [readthedocs.org](https://readthedocs.org/projects/bok-choy/).
That's where I had thought people were getting them from.

I'll build a test version and verify that they come out correctly, so we can get this merged in!

**Comment by sarina** [comment]
@jzoldak thanks - I'm going to go ahead and assign this review to you, then.

**Comment by jzoldak** [comment]
@sarina the changes look good to me on [a doc-edits version I built on readthedocs](http://bok-choy.readthedocs.org/en/doc-edits/). I'll remove that version after we publish latest.
@billagee thanks very much for the edits and clarifications, really appreciate it.

:+1:

**Comment by jzoldak** [comment]
oh also FYI @billagee @sarina we're planning on publishing a new version of bok-choy (v0.4.0) probably next week, which will include the new accessibility testing feature we've been working on. These changes will be picked up in [latest on readthedocs](http://bok-choy.readthedocs.org/en/latest) then.

**Comment by jzoldak** [comment]
@billagee I am getting ready to publish the new release and just noticed that you are not in the AUTHORS file. If you want to be, can you please submit a PR that adds your info to the bottom of it? Thx.

**Comment by billagee** [comment]
@jzoldak Just submitted a PR to add myself to the AUTHORS file - thanks!

**Comment by jzoldak** [comment]
@billagee cool, thx.
I just published v0.4.0 and it's now up on pypi and readthedocs.
We also added a couple new features - a11y testing and visual diff
Check out the CHANGELOG if you're interested.

**Pull request merged**